cmd/kubeadm/app/preflight/checks.go

```diff
@@ -404,13 +404,13 @@ func TryStartKubelet() {
 	// If we notice that the kubelet service is inactive, try to start it
 	initSystem, err := initsystem.GetInitSystem()
 	if err != nil {
-		fmt.Println("[preflight] No supported init system detected, won't ensure kubelet is running.")
+		fmt.Println("[kubelet] No supported init system detected, won't ensure kubelet is running.")
 	} else if initSystem.ServiceExists("kubelet") && !initSystem.ServiceIsActive("kubelet") {
 
-		fmt.Println("[preflight] Starting the kubelet service")
+		fmt.Println("[kubelet] Starting the kubelet service")
 		if err := initSystem.ServiceStart("kubelet"); err != nil {
-			fmt.Printf("[preflight] WARNING: Unable to start the kubelet service: [%v]\n", err)
-			fmt.Println("[preflight] WARNING: Please ensure kubelet is running manually.")
+			fmt.Printf("[kubelet] WARNING: Unable to start the kubelet service: [%v]\n", err)
+			fmt.Println("[kubelet] WARNING: Please ensure kubelet is running manually.")
 		}
```

Comment:
This change doesn't make sense to me.

And honestly, this function doesn't even belong here. @luxas where can we move this? `utils/kubelet`? We can have a check here that returns an error but not kubelet or any other binary/service execution code.